sql/src/main/java/io/crate/metadata/Schemas.java

```diff
@@ -307,19 +404,37 @@ protected void doClose() {
     }
 
     /**
-     * @throws SchemaUnknownException if the view was not found and no such schema exists
+     * @throws RelationUnknown if the view cannot be resolved against the search path.
      */
-    @Nullable
-    public ViewMetaData resolveView(RelationName relationName) {
+    // TODO don't like this return type
```

Comment:
@mfussenegger the ViewMetaData is nested under the view name when streaming so I'm assuming that's why the name wasn't included? Do you think we should revise that or work with a Tuple here?

Reply:
Given the scope of the PR wouldn't change that for now.